invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/footer.less

```diff
@@ -6,26 +6,31 @@
  * under the terms of the MIT License; see LICENSE file for more details.
  */
 
- #footer {
-   color: #fff;
-   margin-top: auto;
+.footer-global {
+  color: #ffffff;
```

Comment:
maybe define also here and a variable e.g `footerTextColor` and set it in the `variables.less`? Similar to what you've done for the `footerDarkColor`...